doc/classes/Curve2D.xml

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="UTF-8" ?>
+﻿<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
Whatever program you used did some sneaky reformatting in this line for both Curve2D and Curve3D which causes the checks to fail. They need to be reverted.